core/src/main/java/org/apache/calcite/plan/Strong.java

```diff
@@ -89,6 +89,10 @@ public static boolean isNotTrue(RexNode node, ImmutableBitSet nullColumns) {
   public static Policy policy(SqlKind kind) {
     return Objects.requireNonNull(MAP.get(kind), kind.toString());
   }
+  /** Just like the policy(kind) method,but the return can be null. */
+  public static Policy policyNullable(SqlKind kind) {
+    return MAP.get(kind);
```

Comment:
Technically speaking, I would rather use existing method, and avoid it failing with NPE in case of failure.
I think `Policy.AS_IS` should be used as a default `Policy`

Reply:
Yes,I agree with you.I will change the default value of policy to AS_IS.